### ISSUE TITLE
Incorrect usage of Etc/... timezones 🤯

### ISSUE THREAD

**Issue by sashabaranov** [tkuchiki/go-timezone#15]
Currently every `GMT+X` abbreviation is converted to `Etc/GMT+X`, which as turns out to be incorrect (kinda shocked by this tbh).

According to [list of tz database time zones](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones) `Etc/GMT+X` is equal to **minus** X hours from GMT. That's right, you can check for yourself. Real `GMT+3` and `Etc/GMT+3` differ by 6 hours, while `GMT+3` and `Etc/GMT-3` are the same.

Wikipedia notes that 

> Sign is intentionally inverted. See the Etc area description.

And in Etc area description:

>Area is the name of a continent, an ocean, or "Etc". The continents and oceans currently used are Africa, America, Antarctica, Arctic, Asia, Atlantic, Australia, Europe, Indian, and Pacific.

> The oceans are included since some islands are hard to connect to a certain continent. Some are geographically connected to one continent and politically to another. See also Boundaries between continents.

> The special area of "Etc" is used for some administrative zones, particularly for "Etc/UTC" which represents Coordinated Universal Time. **In order to conform with the POSIX style, those zone names beginning with "Etc/GMT" have their sign reversed from the standard ISO 8601 convention.** In the "Etc" area, zones west of GMT have a positive sign and those east have a negative sign in their name (e.g "Etc/GMT-14" is 14 hours ahead of GMT.)

**Comment by sashabaranov** [comment]
Besides merged #16 the following is also needed to close this:

- [ ] Check that other uses of Etc/ (in non-GMT abbreviations) are correct
- [ ] Check that offsets in all cases are correct

**Comment by tkuchiki** [comment]
I checked, it looks fine, so that I will close this issue for now.
If you have any problems, please re-open this issue.

https://play.golang.org/p/huUoSWbO-sD

```console
## Etc/GMT+N
Etc/GMT+0 0
Etc/GMT+1 -3600
Etc/GMT+2 -7200
Etc/GMT+3 -10800
Etc/GMT+4 -14400
Etc/GMT+5 -18000
Etc/GMT+6 -21600
Etc/GMT+7 -25200
Etc/GMT+8 -28800
Etc/GMT+9 -32400
Etc/GMT+10 -36000
## Etc/GMT-N
Etc/GMT-0 0
Etc/GMT-1 3600
Etc/GMT-2 7200
Etc/GMT-3 10800
Etc/GMT-4 14400
Etc/GMT-5 18000
Etc/GMT-6 21600
Etc/GMT-7 25200
Etc/GMT-8 28800
Etc/GMT-9 32400
Etc/GMT-10 36000
Etc/GMT-11 39600
Etc/GMT-12 43200
Etc/GMT-13 46800
Etc/GMT-14 50400
```
